WcaOnRails/app/controllers/registrations_controller.rb

```diff
@@ -66,7 +66,7 @@ def psych_sheet_event
 
   def index
     @competition = competition_from_params
-    @registrations = @competition.registrations.accepted.includes(:user, :registration_events)
+    @registrations = @competition.registrations.accepted.includes(:user, :registration_events).order("users.name")
```

Comment:
I've tried this line in console measuring the time with index and without, and it doesn't seem to speed up things.